docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -328,19 +328,21 @@ org_attribute_path = info.roles
 org_mapping = org_foo:org_foo:Viewer org_bar:org_bar:Editor *:org_baz:Editor
 ```
 
-### Configure team synchronization
+## Configure group synchronization
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud/).
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise) and [Grafana Cloud](/docs/grafana-cloud/).
+{{< /admonition >}}
 
-By using Team Sync, you can link your OAuth2 groups to teams within Grafana. This will automatically assign users to the appropriate teams.
-Teams for each user are synchronized when the user logs in.
+Grafana supports synchronization of OAuth2 groups with Grafana teams and role-based access control roles. This allows automatically assigning users to the appropriate teams or automatically granting them the mapped roles.
```

Comment:
> assigning users to the appropriate teams or automatically granting them the mapped roles.

is the feature different for each provider? in that we can map to teams or roles. or is it that we can map to teams and roles as well here.

Reply:
The latter - it's not different between providers, but we do have provider specific setup guides, so I've appended this info to all of them.